src/main/resources/db/migration/V11__add_status_column_to_offers.sql

```diff
@@ -0,0 +1,7 @@
+alter table accommodation_offer add column status text;
```

Comment:
you can add default value here otherwise existing offers will have invalid values

Reply:
I'm not an expert but I wanted to ensure that with spring here:
https://github.com/coi-gov-pl/pomocua-ogloszenia/pull/53/files#diff-b480c2a1c92e979ab8c9358c13f47988b513436bd1dd26a3571d0533da5ac145R61

Reply:
Maybe it's better to add migration change to exiting offers? WDYT?

Reply:
Yeah, that will ensure that the value is present in Java object, but we also don't want to have nulls for that column in the database :)

Reply:
I've added the default value for columns.